src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public void register(@RequestBody UserRequestDto requestDto) {
```

Comment:
I think this method should return UserResponseDto. It's up to you

Reply:
yes, totally makes sense

Reply:
Actually, it is a good practice to always return some data after the creation operations